### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Phani1609** [meta-llama/llama-recipes#414]
### System Info

PyTorch version:2.2.1

### Information

- [X] The official example scripts
- [X] My own modified scripts

### 🐛 Describe the bug

The pretraining loss curve in LLaMA2 appears significantly smoother compared to LLaMA, which shows spikes in loss.

### Error logs

N/A

### Expected behavior

The pretraining loss curve for LLaMA2 should reflect the training process accurately, showing any fluctuations or spikes in loss similar to those seen in LLaMA.

**Comment by mreso** [comment]
Closing this as not relevant to llama-recipes. Moved [this issue](https://github.com/meta-llama/llama/issues/1117) to meta-llama/llama for discussion.
